### ISSUE TITLE
seldon-batch-processor on seldon-core-s2i-python37 image is not generating any output

### ISSUE THREAD

**Issue by omerfsen** [SeldonIO/seldon-core#2745]
## Describe the bug

When running Batch Helm chart i realized that it uses  `image: seldonio/seldon-core-s2i-python37:1.3.0-dev`  i change the value to 1.3.0 or 1.4.0 but after that outputs producted 0 bytes text (So no output)

https://github.com/SeldonIO/seldon-core/blob/master/examples/batch/argo-workflows-batch/helm-charts/seldon-batch-workflow/values.yaml

I changed it to 1.5.0 and it worked fine. 

So `seldonio/seldon-core-s2i-python37:1.3.0` and `seldonio/seldon-core-s2i-python37:1.4.0` has some issues. Also there was no debug output.

## To reproduce

1. Use seldon-core-s2i-python37:1.3.0 or seldon-core-s2i-python37:1.4.0 on https://github.com/SeldonIO/seldon-core/blob/master/examples/batch/argo-workflows-batch/helm-charts/seldon-batch-workflow/values.yaml#L18
2. Run it and you will see 4th step (process-batch-inputs) is taking less then 1 second and output.txt is 0 bytes. 
3. Use seldonio/seldon-core-s2i-python37:1.3.0-dev or seldonio/seldon-core-s2i-python37:1.5.0 and it works fine.

## Model Details 
* Images of your model: [Output of: `kubectl get seldondeployment -n <yourmodelnamespace> <seldondepname> -o yaml | grep image:` where `<yourmodelnamespace>`]
* Logs of your model: [You can get the logs of your model by running `kubectl logs -n <yourmodelnamespace> <seldonpodname> <container>`]

**Comment by omerfsen** [comment]
When there is an issue:

argo logs -n emtech seldon-batch-process

```
argo logs -n emtech  seldon-batch-process 
seldon-batch-process-3626514072: time="2020-12-04T12:28:56.505Z" level=info msg="Starting Workflow Executor" version=v2.11.7
seldon-batch-process-3626514072: time="2020-12-04T12:28:56.509Z" level=info msg="Creating a docker executor"
.....
seldon-batch-process-3626514072: time="2020-12-04T12:28:56.510Z" level=info msg="kubectl create -f /tmp/manifest.yaml -o json"
seldon-batch-process-3626514072: time="2020-12-04T12:28:58.601Z" level=info msg=emtech/SeldonDeployment.machinelearning.seldon.io/sklearn
seldon-batch-process-3626514072: time="2020-12-04T12:28:58.603Z" level=info msg="Starting SIGUSR2 signal monitor"
seldon-batch-process-3626514072: time="2020-12-04T12:28:58.603Z" level=info msg="No output parameters"
seldon-batch-process-2052519094: I1204 12:29:09.816215       9 request.go:621] Throttling request took 1.1716588s, request: GET:https://10.12.16.1:443/apis/templates.gatekeeper.sh/v1alpha1?timeout=32s
seldon-batch-process-2052519094: I1204 12:29:13.087703      16 request.go:621] Throttling request took 1.14631s, request: GET:https://10.12.16.1:443/apis/apiextensions.k8s.io/v1beta1?timeout=32s
seldon-batch-process-2052519094: Waiting for deployment "sklearn-default-0-classifier" rollout to finish: 0 of 3 updated replicas are available...
seldon-batch-process-2052519094: Waiting for deployment "sklearn-default-0-classifier" rollout to finish: 1 of 3 updated replicas are available...
seldon-batch-process-2052519094: Waiting for deployment "sklearn-default-0-classifier" rollout to finish: 2 of 3 updated replicas are available...
seldon-batch-process-2052519094: deployment "sklearn-default-0-classifier" successfully rolled out
seldon-batch-process-1257652469: Added `minio-local` successfully.
seldon-batch-process-1257652469: `minio-local/data/input-data.txt` -> `/assets/input-data.txt`
seldon-batch-process-1257652469: Total: 0 B, Transferred: 146.48 KiB, Speed: 8.96 MiB/s
seldon-batch-process-2033515954: Elapsed time: 0.002772092819213867
seldon-batch-process-2123074048: Added `minio-local` successfully.
seldon-batch-process-2123074048: `/assets/output-data.txt` -> `minio-local/data/output-data-a2a328c5-92fd-4bd7-964c-089eb8fb8323.txt`
seldon-batch-process-2123074048: Total: 0 B, Transferred: 0 B, Speed: 0 B/s
```


output-data-uid.txt is 0 bytes. and Elapsed time is less then 1 seconds..

**Comment by axsaucedo** [comment]
@omerfsen are we good to close this after merging #2746?

**Comment by omerfsen** [comment]
@axsaucedo if those image versions (seldonio/seldon-core-s2i-python37:1.3.0 and seldonio/seldon-core-s2i-python37:1.4.0) are not used elsewhere that is fine you can close. I know it is already fixed for 1.5.0 so if your support policiy is latest minor versions only please close this ticket too.
